swap/tests/testutils/mod.rs

```diff
@@ -107,7 +106,12 @@ pub struct TestContext {
 }
 
 impl TestContext {
-    pub async fn new_swap_as_bob(&mut self) -> (bob::Swap, BobEventLoopJoinHandle) {
+    pub async fn alice_swap_triggered_by_bob(&mut self) -> alice::Swap {
+        // return the swap that is triggered by bob
```

Comment:
This comment does not add anything the function name doesn't already say :)

Reply:
nice spot, I actually had renamed the function and then forgot to remove the comment. good all rule: don't add a comment before you have not checked if you can fix it by making the code around better...